app/helpers/self_service/breadcrumbs_helper.rb

```diff
@@ -62,5 +66,9 @@ def crumb_firm_adviser_new
     def crumb_trading_name_new
       { name: t('self_service.trading_name_new.breadcrumb', firm_name: @firm.registered_name) }
     end
+
+    def crumb_principal_edit
+      { name: t('self_service.principal_edit.breadcrumb', firm_name: @firm.registered_name) }
+    end
   end
 end
```

Comment:
Yeah I wouldn't say "happy" necessarily, but I'd say Caden is right, this is a separate issue that shouldn't block this PR.